vector.go

```diff
@@ -134,7 +136,9 @@ func (vec *vector) getChildVectors(v mapping.Vector, writable bool) {
 	switch vec.Type {
 	case TYPE_LIST, TYPE_MAP:
 		child := mapping.ListVectorGetChild(v)
-		vec.childVectors[0].initVectors(child, writable)
+		if child.Ptr != nil {
+			vec.childVectors[0].initVectors(child, writable)
+		}
```

Comment:
How do you trigger `child.Ptr` being `nil`? That seems like unintentional behaviour?

Reply:
Good point. I removed the nil checks.